### ISSUE TITLE
kitti_data/train.txt file is missing

### ISSUE THREAD

**Issue by ghimiremukti** [sshleifer/object_detection_kitti#4]
following the tutorial (https://medium.com/@sshleifer/how-to-finetune-tensorflows-object-detection-models-on-kitti-self-driving-dataset-c8fcfe3258e9), object_detection_kitti/object_detection/kitti_data directory which further contains training and testing images and labels as below

![capture](https://user-images.githubusercontent.com/13733258/38283752-c8347b0e-3785-11e8-88e6-983f240a0787.JPG)

When run    
$ cd object_detection_kitti/object_detection
$ python create_dataset.py 

it throws an error:

 File "create_dataset.py", line 182, in <module>
    do_kitti_ingest()
  File "/home/train/anaconda2/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/train/anaconda2/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/train/anaconda2/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/train/anaconda2/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "create_dataset.py", line 173, in do_kitti_ingest
    VOC_TRAIN_DIR, os.path.join(data_dir, 'train.txt'))
  File "/home/train/tftrain/object_detection_kitti/object_detection/kitti_to_voc.py", line 15, in kitti_to_voc
    train_ids=train_id_path)
  File "/home/train/tftrain/object_detection_kitti/object_detection/vod_converter/converter.py", line 136, in convert
    image_detections = ingestor.ingest(from_path, train_ids)
  File "/home/train/tftrain/object_detection_kitti/object_detection/vod_converter/kitti.py", line 64, in ingest
    image_ids = self._get_image_ids()
  File "/home/train/tftrain/object_detection_kitti/object_detection/vod_converter/kitti.py", line 85, in _get_image_ids
    with open(self.train_id_path) as f:
IOError: [Errno 2] No such file or directory: u'kitti_data/train.txt'

it seems "kitti_data/train.txt" file should be there. where is this file coming from? or if $ python create_dataset.py creates it, then why it is asked beforehand.

**Comment by ghimiremukti** [comment]
kitti_data
└────training
| └────image_2
| | ├── 00000.jpg
| | ├── 00001.jpg
| | └── ....jpg
| └────label_2
| ├── 00000.txt
| ├── 00001.txt
| └── ....txt
└────testing
└──image_2
├── 00000.jpg
├── 00001.jpg
└── ....jpg

**Comment by ghimiremukti** [comment]
I am hoping someone is looking at this issue I faced :-(

**Comment by sshleifer** [comment]
Good catch! If you `git pull` create_dataset should create train.txt as a side effect, I believe

**Comment by ghimiremukti** [comment]
Hi @sshleifer , thanks for the reply. Now could you tell how can we generate it, before running $ python create_dataset.py ?

**Comment by ghimiremukti** [comment]
after adding 'kitti_data/train.txt' manually, another error pops up:

.....
.....
.....
File "/home/train/tftrain/object_detection_kitti/object_detection/vod_converter/kitti.py", line 102, in _get_detections
    with open(detections_fpath) as f:
IOError: [Errno 2] No such file or directory: u'/home/train/tftrain/object_detection_kitti/object_detection/kitti_data/training/label_2/.txt'


I think just by adding 'kitti_data/train.txt' didn't solve the issue means the error is somewhere deeper.

Please try running user@...:object_detection_kitti/object_detection$ python create_dataset.py, and you will see the error. Thanks.

**Comment by sshleifer** [comment]
you can either run create_dataset again
or you can read the code

**Comment by ghimiremukti** [comment]
I did run $run create_dataset.py several times, but no luck.

**Comment by sshleifer** [comment]
works on my machine, feel free to fix and send a pull request!
